### PR TITLE
Api calls

### PR DESCRIPTION
**Pull request by mmsesay**
In the PR I did the following:

- Add event listener to submit the form and to refresh div container
- Add a file to make the API request for:
     - creating a game
     - creating a game score based on the arguments (user and score)
     - fetch all the game scores
- Add helper function and methods to hide and show a message box, render scores and  reset input fields

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -1 +1,28 @@
...
+
+  const resetInputFields = (args) => {
+    args.forEach((element) => {
+      element.value = '';
+    });
+  };
+
+  submitButton.addEventListener('click', (e) => {
+    e.preventDefault();
+    fetcher.createGameScore(nameField.value, scoreField.value);
```
Comment (EricMbouwe) [inline src/js/index.js]: - [x] `Name` and `score` fields should be properly validated to ensure the user cannot submit blank entries. If you try and check your console you will see this

Screenshot from 2021-10-06 13-52-49

Let's fix that.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi STUDENT,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR
- Good commits
- Good code organization

# Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/index.js (hunk 1 above)
Comment (mmsesay) [reply src/js/index.js]: Thanks for the suggestion. I have added the validation.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Muhammad, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer(s). There is nothing else to say other than... it's time to merge it 🍾 :shipit:.
Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**